src/arrays/referencing_array_and_its_elements/hard/q2/InThirds.java

```diff
@@ -0,0 +1,40 @@
+package arrays.referencing_array_and_its_elements.hard.q2;
+
+/*
+Write a program called InThirds that defines an array of integers, in this case {3, 4, 6, 21, 48, 42, 89, 2, 0}, and
```

Comment:
This question is bad and going to be hard to test thoroughly. Change it to:
`Write a Java program that takes an array of nine numbers from the user and a positive integer x. Then, call a method named remainder(int[] arr, int x) that takes the array and returns the first n numbers in a new array of length n, where n is the remainder after dividing the array length by x. Print the new array in the main method. Assume the user provides valid input. `